templates/openio.pp.j2

```diff
@@ -40,7 +40,7 @@ openiosds::oioeventagent {'oio-event-agent-1':
   {% if openio_oioeventagent_webhooks is defined -%}
   webhooks => {{ openio_oioeventagent_webhooks }},
   {% endif -%}
-  {% if openio_replication_queue_url -%}
+  {% if openio_replicator == 1 -%}
```

Comment:
`== 1 ?`
Prefer use a boolean
When `openio_replication_queue_url` is non empty, it is `true`
A valid enhancement could be
`{% if openio_replication_queue_url is defined and openio_replication_queue_url -%}`

Reply:
openio_replication_queue_url is set by default on playbook openiosds.yml, consequently is always true.
(openio_replication_queue_url: "beanstalk://{{ openio_bind_address ~ ':6014' if openio_replicator else '' }}")
consequently, on each deployment the replicator is enable.
I ll check with boolean, but ok if you prefer to use your enhancement.